BlueprintUI/Sources/Layout/ConstrainedSize.swift

```diff
@@ -82,6 +82,23 @@ extension ConstrainedSize {
                 return absoluteValue
             }
         }
+
+        public static func == (lhs: Self, rhs: Self) -> Bool {
```

Comment:
Do you need this? Shouldn't the compiler synthesize it automatically since all the cases contain equatable values?

Reply:
Oh hmm. I got endless complaints when adding it via an extension in SPOS but I didn't try here to see if it differed. (I didn't think Equatable provided automatic conformance when you had associated values)

Reply:
Yup works great. Weird.